samples/deployment/aws/deploy_and_destroy.yaml

```diff
@@ -0,0 +1,130 @@
+- name: "My AWS infrastructure"
+  hosts: localhost
+  gather_facts: no
+  vars:
+    ftd_build_id: "6.4.0-1304"
+    ec2_instance_type: "c4.xlarge"
+    ansible_ssh_private_key_name: "ngfwv-prod-temp-kp.pem"
+    ansible_ssh_private_key_file: "./ngfwv-prod-temp-kp.pem"
+  tasks:
+    # Prepare Env Section
+    - name: "Create prod key"
```

Comment:
Why do we have `prod` here?

Reply:
changed name to "Create production key pair" - just to distinguish key pairs between playbooks.